.gitignore

```diff
@@ -0,0 +1,7 @@
+deploy_key
+packrat/lib*/
+
+# Dont put local caches or builds into git
+site/*_cache
+site/*_files
+sites/docs/*
```

Comment:
`site/docs/`

Reply:
Corrected in d3f92a4d179920cacfa86fd409684c52b2447ffd